### PR TITLE
Record FxA Verified events in SFDC

### LINKED ISSUE #149: Put FxA user data into SFDC in addition to SFMC
FxA users will soon need to go into SFDC instead of the data extension in SFMC into which they're going now. The timing for this will need to be coordinated with LCM and the IT Salesforce folks.

### PR DESCRIPTION
**Pull request by pmac**
Also continue to record them in SFMC.

Fix #149

### CODE REVIEW

**Comment by pmac** [comment]
@andrewmozilla I've added a switch in the code so that we can go ahead to prod (hopefully tomorrow) and then only enable the flow of data to SFDC when we're ready.

File: basket/news/tasks.py [hunk 1]
```diff
@@ -1350,3 +1346,34 @@ def snitch(start_time=None):
     requests.post('https://nosnch.in/{}'.format(snitch_id), data={
         'm': totalms,
     })
+
+
+def get_fxa_user_data(fxa_id, email):
```
Comment (glogiotatidis) [inline basket/news/tasks.py]: Does it make sense to use this function here too?

https://github.com/mozmeao/basket/blob/a1df42d257d0b8bb63c41882eabd05b24adf7fe3/basket/news/tasks.py#L749

File: basket/news/tasks.py [hunk 2]
```diff
@@ -741,28 +758,7 @@ def process_subhub_event_customer_created(data):
...
-            sfdc.update(user_data_fxa, {
-                'fxa_id': f"DUPE:{contact_data['fxa_id']}",
-                'fxa_deleted': True,
-            })
-
-    # if we still don't have user data try again with email this time
-    if not user_data:
-        user_data = get_user_data(email=data['email'], extra_fields=['id'])
-
+    user_data = get_fxa_user_data(data['user_id'], data['email'])
```
Comment (pmac) [inline basket/news/tasks.py]: @glogiotatidis  I did change that out here (re: your comment on DRY)

**Comment by pmac** [comment]
Thanks for the quick review as always @glogiotatidis 🥇

**Pull request merged**